### ISSUE TITLE
On Neural Network Identification for Low-Speed Ship Maneuvering Model. (arXiv:2111.06120v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#13047]
On Neural Network Identification for Low-Speed Ship Maneuvering Model. (arXiv:2111.06120v1 [eess.SY])<br>
https://ift.tt/3n7EuLt<br>
<p>Several studies on ship maneuvering models have been conducted using captive model tests or computational fluid dynamics (CFD) and physical models, such as the maneuvering modeling group (MMG) model. A new system identification method for generating a low-speed maneuvering model using recurrent neural networks (RNNs) and free running model tests is proposed in this study. We especially focus on a low-speed maneuver such as the final phase in berthing to achieve automatic berthing control. Accurate dynamic modeling with minimum modeling error is highly desired to establish a model-based control system. We propose a new loss function that reduces the effect of the noise included in the training data. Besides, we revealed the following facts - an RNN that ignores the memory before a certain time improved the prediction accuracy compared with the "standard" RNN, and the random maneuver test was effective in obtaining an accurate berthing maneuver model. In addition, several low-speed free running model tests were performed for the scale model of the M.V. Esso Osaka. As a result, this paper showed that the proposed method using a neural network model could accurately represent low-speed maneuvering motions.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
